quickstarts/microsoft.web/function-app-create-dedicated/metadata.json

```diff
@@ -5,16 +5,39 @@
   "description": "This template provisions a function app on a dedicated hosting plan, meaning it will be run and billed just like any App Service site.",
   "summary": "Provision a function app on a dedicated hosting plan",
   "githubUsername": "mattchenderson",
-  "dateUpdated": "2021-05-12",
+  "dateUpdated": "2022-07-25",
   "tags": {
     "subscriptionType": "Azure4Student",
+    "id": 18,
+    "rank": 4,
     "cost": "$$",
     "resource": "App Service, Function",
     "level": "Intermediate",
     "scenario": "Azure Function",
     "roles": "Developer",
-    "learning": "https://docs.microsoft.com/en-us/learn/paths/build-workflows-with-logic-apps/?WT.mc_id=Portal-Microsoft_Azure_Education",
-    "scenarioLink": "https://azure.microsoft.com/en-us/services/logic-apps/",
-    "isActive": "true"
+    "learning": "https://docs.microsoft.com/en-us/learn/browse/?products=azure&term=cognitive&WT.mc_id=Portal-Microsoft_Azure_Education&terms=cognitive",
+    "isA4EActive": "true",
+    "usefulLink": [
+            {
+                "name": "Getting Started",
+                "rank": 1,
+                "url": "https://azure.microsoft.com/en-us/resources/templates/function-app-create-dedicated/"
```

Comment:
https://azure.microsoft.com/en-us/resources/templates/ is being deprecated, you'll want to reference samples here: https://docs.microsoft.com/en-us/samples/browse/?expanded=azure&products=azure-resource-manager

https://docs.microsoft.com/en-us/samples/azure/azure-quickstart-templates/function-app-create-dedicated/